common/determined_common/storage/s3.py

```diff
@@ -1,8 +1,8 @@
+import contextlib
```

Comment:
non-blocking: This isn't within scope for this PR but the GCS API is S3 compliant. There may be a way to combine the two storage managers.

Reply:
ok